src/web/crate_details.rs

```diff
@@ -182,7 +188,10 @@ impl CrateDetails {
             default_target: krate.get("default_target"),
             doc_targets: MetaData::parse_doc_targets(krate.get("doc_targets")),
             yanked: krate.get("yanked"),
-            rustdoc_css_file: get_correct_docsrs_style_file(krate.get("doc_rustc_version"))?,
+            rustdoc_css_file: krate
```

Comment:
related question: 
was there a time when we only stored records in the `releases` table? and nothing in `builds`? 

Wouldn't this change not break old docs?

Reply:
If so that was before my time, would have to check the prod database dump